new_core/screens/initial.js

```diff
@@ -57,7 +57,7 @@ export function render (canvas, ctx) {
   ctx.fillRect(0,0,canvas.width,canvas.height)
 
   // This draws the image onto the screen
-	ctx.drawImage(jslix.getImg(0), 90, 120, 200, 20)
+	ctx.drawImage(jslix.getImg(0), SCREEN_WIDTH - 210, SCREEN_HEIGHT - 30, 200, 20)
   //ctx.drawImage(jslix.quickImage("../image/mobile/cwttitle.png"), 10, 120, 200, 20)
```

Comment:
no need to fear the deletion of comments.  the code is persisted in the history, so in my opinion there is no reason to leave it as comment in the code.

see: http://geekswithblogs.net/MarkPearl/archive/2011/09/12/commenting-code-is-an-anti-pattern.aspx

Reply:
Yeah, I'll edit this in another commit to this branch to get rid of these extra bits of information. I thought I caught them all, but this must have came back when I merged the code with master :(

Reply:
no problem 😁